includes/class-sensei-updates.php

```diff
@@ -189,6 +190,26 @@ public function __construct( $parent ) {
 
 	} // End __construct()
 
+	/**
+	 * Get the array of upgrades that have executed.
+	 *
+	 * @return array
+	 */
+	public function get_ran_upgrades_raw() {
+		$upgrades = get_option( $this->token . '-upgrades', array() );
+
+		if ( empty( $upgrades ) && $this->token_legacy && get_option( $this->token_legacy . '-upgrades', false ) ) {
+			$upgrades = get_option( $this->token_legacy . '-upgrades', false );
```

Comment:
I have the same comment here as https://github.com/Automattic/sensei/pull/2448#discussion_r257349785 regarding the two `get_option` calls.

Reply:
Shouldn't hit database twice, both auto-loaded, but I can refactor. e3e82feee86a100be2146478ad3a88081a5a76f2